syntax="proto3";

import "user.proto";

package fixbug;
option cc_generic_services=true;

message GetFriendsListRequest{
    uint32 userid=1;
}
message GetFriendsListResponse{
    ResultCode result=1;
    repeated bytes friends=2;
}

service FriendsServiceRpc{
    rpc GetFriendsList(GetFriendsListRequest)returns(GetFriendsListResponse);
}
